notebooks/From FHIR to ML - Getting Started with the PHC Python SDK.ipynb

```diff
@@ -0,0 +1,2264 @@
+{
```

Comment:
<p>typo: be <em>an</em> continuous integer feature</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lifeomic/phc-examples/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>